src/category_theory/closed/zero.lean

```diff
@@ -0,0 +1,66 @@
+/-
+Copyright (c) 2020 Bhavik Mehta. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Bhavik Mehta
+-/
+
+import category_theory.closed.cartesian
+import category_theory.limits.shapes.zero
+import category_theory.punit
+import category_theory.conj
+
+/-!
+# A cartesian closed category with zero object is trivial
+
+A cartesian closed category with zero object is trivial: it is equivalent to the category with one
+object and one morphism.
+
+## References
+
+* https://mathoverflow.net/a/136480/117945
```

Comment:
The last part of the URL is actually your own MO user id, see e.g. [this meta MO discussion](https://meta.mathoverflow.net/questions/2787/remove-the-user-identifying-part-of-the-share-url).
```suggestion
* https://mathoverflow.net/a/136480
```